server/labs/index.js

```diff
@@ -0,0 +1,114 @@
+var EMPTY = 0,
+  FILL = 1;
+
+module.exports = {
+  getLab: function(n, m) {
+    var arr = new Array(n),
+    filledLines = [],
+    emptyLines = [];
+
+    arr[0] = new Array(m);
+    arr[n - 1] = new Array(m);
+
+    for (var i = 0; i < m; ++i) {
```

Comment:
I would rename i => x and  j => y or vice versa 
x will iterate over width
y will iterate over height